libamqpprox/amqpprox_server.cpp

```diff
@@ -312,5 +315,10 @@ boost::asio::ssl::context &Server::egressTlsContext()
     return d_egressTlsContext;
 }
 
+boost::asio::io_service &Server::ioService()
```

Comment:
can we remove this? looks like its not being used, not sure why we'd need to expose it.

Reply:
We need this function to configure auth service using control command in future. But yeah for the current PR, we can remove this function.